src/drivers/abstract/attribute.js

```diff
@@ -11,6 +11,9 @@ const { snakeCase } = require('../../utils/string');
 function findJsType(DataTypes, type, dataType) {
   if (type instanceof DataTypes.BOOLEAN) return Boolean;
   if (type instanceof DataTypes.JSON) return JSON;
+  if (type instanceof DataTypes.BINARY || type instanceof DataTypes.VARBINARY || type instanceof DataTypes.BLOB) {
```

Comment:
`type instanceof DataTypes.VARBINARY` won't be necessary since `type instanceof DataTypes.BINARY` already has this condition covered.